### PR TITLE
docs: fix bash command and version used in tutorial

### PR DESCRIPTION
**Pull request by abusaidm**
bash command that contain regex needs to be quoated, version doesn't exists

Before creating your pull request, make sure that you have
[signed the DOC](https://github.com/ory/hydra/blob/master/CONTRIBUTING.md#developers-certificate-of-origin). You can ammend your
signature to the current commit using `git commit --amend -s`.

Please create PRs only for bugs, or features that have been discussed with the maintainers, either at the
[ORY Community](https://community.ory.am/) or join the [ORY Chat](https://gitter.im/ory-am/hydra).

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ $ docker run -d \
   -e ISSUER=https://localhost:9000/ \
   -e CONSENT_URL=http://localhost:9020/consent \
   -e FORCE_ROOT_CLIENT_CREDENTIALS=admin:demo-password \
-  oryd/hydra:v0.10.0-alpha1
+  oryd/hydra:v0.10.0-alpha.1
```
Comment (arekkas) [inline docs/install.md]: Could you please change this to `0.10.0-alpha.8`? Thank you! :)

Comment (abusaidm) [reply docs/install.md]: is `v0.9.12` the latest stable version?

**Comment by abusaidm** [comment]
done.

**Comment by arekkas** [comment]
Thank you!

**Pull request merged**